src/views/Title/Title.tsx

```diff
@@ -18,9 +20,19 @@ interface LoaderData {
   preferred_username?: string
   groups?: string[]
 }
+const emptyUser: LoaderData = {
+  email: '',
+  name: '',
+  preferred_username: '',
+  groups: [],
+}
 const Title: React.FC = (): JSX.Element => {
   const loaderData = useLoaderData() as string
-  const userInfo = jwtDecode(loaderData) as LoaderData
+  const userInfo: LoaderData =
+    loaderData === 'unauthorized\n'
```

Comment:
Can you check against the response status instead? If it is not 200 then its an error